flixel/tweens/FlxTween.hx

```diff
@@ -735,3 +853,15 @@ class FlxTweenManager extends FlxBasic
 		}
 	}
 }
+
+class ThenCommand
```

Comment:
Also, there's not really any reason to expose this API to the user, would mean we have to worry about backwards compatibilty etc. Should be a `private class`.